app/assets/stylesheets/global.css.sass

```diff
@@ -2,169 +2,130 @@
-      line-height: 1em
-      color: $lightgray
-
-    a
-      display: inline-block
-      height: 48px
-      text-decoration: none
-
-li
-  list-style-type: none
-
-
-header
-  h1
-    display: inline-block
-    margin: 0
-    width: 210px
-    height: 58px
-    a
-      display: block
-      height: 100%
-      width: 100%
-      background-image: asset-url('logo_subnavi.png')
-      background-repeat: no-repeat
-      background-size: cover
-      background-position: -15px -1px
-      text-indent: 999%
-      white-space: nowrap
+    list-style-type: none
+  .text-right
+    text-align: right
+
+// For sticky footer
+#wrap
+  min-height: 100%
+  height: auto
+  // Negative indent footer by its height
+  margin: 0 auto -60px
+  // Pad bottom by footer height
+  padding: 0 0 60px
+
+#header
+  height: 60px
+  background-color: $railsred 
+  img
+    width: 205px
+    margin: -3px 0 0 -15px
+
+  a.button-burger
+    @include icon($reorder)
+    width: 35px
+    height: 35px
+    font-size: 24px
+    text-decoration: none
+    color: $railsred
+    background: $white
+    border-radius: 50%
+    overflow: hidden
+    margin-top: 12px
+    @media screen and (min-width: 992px)
+      display: none
+    i
+      position: relative
+      top: 5px
+      right: 7px
+  ul
+    padding: 0
+    margin-top: -15px
+    text-align: right
+    li
+      margin-bottom: 0.5rem
 
-  nav
-    clear: right
-    float: right
-    @include clearfix
+  nav.user
+    color: $darkgray
 
-    &.top
+  nav.top
+    @media screen and (min-width: 992px)
       ul
-        text-align: center
-        position: relative
-        top: 5px
-
+        margin-top: -55px
       li
         display: inline-block
-        margin: 1em 0 0 0
-
-        &:last-child:after
-          content: none
-
         &:after
-          content: ''
-          display: inline-block
-          height: 20px
-          width: 40px
-          background: asset-url('star.png') 16px 8px no-repeat
-
-        a
-          display: inline-block
-          height: 48px
+          content: ''
+          width: 2rem
+          font-family: FontAwesome
+          font-size: 0.7rem
           color: $white
-          text-decoration: none
-
-          &.active
-            text-decoration: underline
-            background-image: asset-url('header_nav_active.png')
-            background-repeat: no-repeat
-            background-position: center 32px
-            text-decoration: none
-
-    &.user
-      font-size: 0.9em
-      color: $gray
-
-nav
-  .back:before
-    content: '\2190'
-    display: inline-block
-    position: absolute
-    margin-left: -1.5em
-    margin-top: -1px
+          margin: 0 0.6rem 0 0.6rem
+        &:last-child:after, &:first-child:after
+          content: none
+      a
+        color: white
+        text-decoration: none
 
-  &.page
-    font-size: 0.9em
+    // Changes bootstraps default collapsing breakpoint
+    @media (min-width: 768px) and (max-width: 991px)
```

Comment:
same as my comment above you can use [$screen-sm](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L300) and [$screen-sm-max](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L321)